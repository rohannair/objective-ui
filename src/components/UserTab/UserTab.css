.UserTab {
  display: flex;
  align-items: center;
}

.avatarContainer {
}

.avatar {
  width: 48px;
  height: 48px;
  border-radius: 64px;
}

.name {
  line-height: 1;
  margin-bottom: 0;
}

.content {
  margin-left: 15px;
}
