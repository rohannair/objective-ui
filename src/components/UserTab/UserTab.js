import React, { PropTypes } from 'react';
import styles from './UserTab.css';
import classNames from 'classnames/bind';

let cx = classNames.bind(styles);

const UserTab = p => {
  const classname = cx({
    [styles.UserTab]: true
  });

  return (
    <div className={classname}>
      <div className={styles.avatarContainer}>
        <img src={p.img} className={styles.avatar} />
      </div>
      <div className={styles.content}>
        <h3 className={styles.name}>{p.firstName} {p.lastName}</h3>
        { p.children }
      </div>
    </div>
  );
};

export default UserTab;
