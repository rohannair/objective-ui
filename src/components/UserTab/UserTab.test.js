import React from 'react'
import UserTab from './index.js'

describe('Test suite for UserTab component', () => {
  it('UserTab should exist', () => {
    let wrapper = shallow(<UserTab/>)
    expect(wrapper).toMatchSnapshot()
  })
})

