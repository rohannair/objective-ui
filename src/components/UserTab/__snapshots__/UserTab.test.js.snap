exports[`Test suite for UserTab component UserTab should exist 1`] = `
<div>
  <div>
    <img />
  </div>
  <div>
    <h3>
       
    </h3>
  </div>
</div>
`;
